assets/pages/new-contributions-landing/components/ContributionPayment.jsx

```diff
@@ -70,23 +75,21 @@ function initialiseStripeCheckout(props: PropTypes) {
     .then((handler: PaymentHandler) => props.isPaymentReady(true, { Stripe: handler }));
 }
 
-const recurringPaymentMethodInitialisers: {[PaymentMethod]: (props: PropTypes) => void} = {
+// Bizarrely, adding a type to this object means the type-checking on the
+// paymentMethodInitialisers is no longer accurate.
+// (Flow thinks it's OK when it's missing required properties).
+const recurringPaymentMethodInitialisers = {
   PayPal: () => { /* TODO PayPal recurring */ },
   Stripe: initialiseStripeCheckout,
-  DirectDebit: () => { /* no initialisation required */ },
```

Comment:
I think you've accidentally deleted `DirectDebit`

Reply:
I actually deleted it because it's covered by the base handlers... but I like the explicitness of the comment so I'll re-instate it